src/test/enginesynctest.cpp

```diff
@@ -47,13 +49,21 @@ class EngineSyncTest : public MockedEngineBackendTest {
                                                       "sync_master"))
                             ->toBool();
         }
-        if (ControlObject::getControl(ConfigKey(group, "sync_mode"))->get() != SYNC_FOLLOWER) {
+        if (auto mode = ControlObject::getControl(ConfigKey(group, "sync_mode"))
+                                ->get();
+                mode != SYNC_FOLLOWER) {
+            qWarning() << "expected mode SYNC_FOLLOWER, got" << mode;
```

Comment:
Are you testing these from cases? Shouldn't this use EXPECT?

Reply:
Same for the other warnings in the diff.

Reply:
No, because then the test failures always show up at line 55 or whatever, making it impossible to determine what comparison actually failed.  by doing EXPECT_TRUE(isFollower()) I see the correct failure location.

Reply:
Ok